Packs/Workday/Integrations/Workday_IAM/Workday_IAM.yml

```diff
@@ -0,0 +1,65 @@
+category: Identity and Access Management
+commonfields:
+  id: Workday IAM
+  version: -1
+configuration:
+- display: Username
+  name: credentials
+  required: false
+  type: 9
+- display: Workday Report URL
+  name: report_url
+  required: true
+  type: 0
+- additionalinfo: Import schedule to get events
+  display: Fetch Events Frequency In minutes
+  name: fetch_events_time_minutes
+  required: false
+  type: 0
+- additionalinfo: Maximum number of events to fetch at a time
+  defaultvalue: '50'
+  display: Fetch Limit (Recommended less than 200)
+  name: max_fetch
+  required: false
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
```

Comment:
add a parameter for first fetch. by default true. 
if set to true then will run the first fetch, if not then will not run first fetch